src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/coding/hiddenfield/Example2.java

```diff
@@ -0,0 +1,34 @@
+/*xml
+<module name="Checker">
+  <module name="TreeWalker">
+    <module name="HiddenField">
+      <property name="tokens" value="VARIABLE_DEF"/>
+    </module>
+  </module>
+</module>
+*/
+package com.puppycrawl.tools.checkstyle.checks.coding.hiddenfield;
+
+// xdoc section -- start
+class Example2 {
+
+  private String field;
+  private String testField;
+
+  Example2(String testField) { // OK, 'testField' param doesn't hide any field
```

Comment:
Remove this comment, it is ok because we don't check parameters.

Reply:
Done replaced it with appropriate comment